uber/templates/preregistration/preregbase.html

```diff
@@ -95,7 +101,9 @@
                 price: {{ c.BADGE_TYPE_PRICES[badge_type] }},
                 badge_type: '{{ badge_type }}',
                 onClick: function () {
-                    $('input[name="badge_type"]').val('{{ badge_type }}');
+                    if ($('input[name="badge_type"]').val() != '{{ c.STAFF_BADGE }}') {
```

Comment:
Yes, I'm sure about that. The extra prereg badge types are specifically for MFF, and if you prevent people from changing their badge type here, they won't be able to upgrade to Sponsor or Shiny Sponsor if they're Staff or Guest. This won't be perfect -- if someone selects Sponsor, selects back to Attendee, and saves, it'll set them back to Attendee -- but I can fix that later and it's a much less serious bug than "can't upgrade."